pure2.md

```diff
@@ -0,0 +1,535 @@
+
+    2.  Check that the Targets metadata from the Image repository matches the offline-update Targets metadata from the Director repository for this image:
+
+        1.  Check that the non-custom metadata (i.e., length and hashes) of the target are the same in both sets of metadata.
+
+        2.  Check that all SHALL match custom metadata fields, as defined in section 5.2.3.1.1 of the Standard, are the same in both sets of metadata.
+
+#### Additional step for full verification during an online update
+
+Any vehicle eligible for offline updates that also receives normal OTA updates SHALL additionally download and check for a newer version of the offline-updates Snapshot metadata from the Director repository when it executes an update cycle:
+
+1.  Download the latest offline-update Snapshot metadata file from the Director repository.
+
+2.  Check that it has been signed by the threshold of unique keys specified for the offline-updates role in the latest Root metadata file. If the new Snapshot metadata file is not signed as required, discard it and log a warning (but continue with the update cycle).
+
+3.  Check that the version number listed by the previous Snapshot metadata file for each Targets metadata file is less than or equal to its version number in this Snapshot metadata file. If this condition is not met, discard the new Snapshot metadata file, abort the update cycle, and log the failure. (Checks for a rollback attack.)
+
+### Addendum to Uptane Standard Build-time prerequisite requirements for ECUs
+
+For an ECU to be capable of receiving PURE-2-secured offline updates, it MUST have the following data provisioned at the time it is manufactured or installed in the vehicle:
+
+1. A sufficiently recent copy of required Uptane metadata at the time of manufacture or install. This is necessary for the ECU to authenticate that the remote repository is legitimate when it first downloads metadata in the field. See [Uptane Deployment Best Practices](https://uptane.github.io/deployment-considerations/index.html) for more information. 
+2. The current time, or a secure attestation of a sufficiently recent time.
+
+The ECU does not require an ECU key in order to be capable of receiving PURE-2-secured offline updates.
+
+### Sample directory structure
+
+Note that, although the offline-update snapshot metadata file lists two possible offline updates (EMEA-standard.json and EMEA-premium.json, see sample metadata below), a single source of offline-update metadata SHOULD always only include one of them--otherwise, the device wouldn't know which one to install. An implementor MAY choose to allow multiple offline-update targets metadata files in the same location, however: for example, if the Uptane client on the primary has the capability to display different options and allow the technician or end-user to select the one they wish to install.
```

Comment:
Maybe this note should be closer to the Snapshot metadata example file? Here it's a bit confusing. I also think the last sentence is structured in a fairly convoluted manner.

Reply:
Addressed both